source/settings/plugin/acl-plugin.rst

```diff
@@ -35,29 +36,71 @@ Settings
      file in each mailbox (or :dovecot_core:ref:`mail_control_path`) directory.
      This is the default.
 
-   This backend has the following options:
 
-   ============== ============================================================
-   Name           Description
-   ============== ============================================================
-   <global_path>  If a path is defined, this is the location of the global ACL
-                  configuration file.
-   cache_secs     The interval, in seconds, for running stat() on the ACL file
-                  to check for changes. DEFAULT: 30 seconds
-   ============== ============================================================
+.. dovecot_plugin:setting:: acl
+   :plugin: acl
+   :added: 3.0.0
+   :values: @named_filter_list
 
-   Example:
+   Specifies an ACL entry on global, namespace or mailbox level.
+
+   Has two settings, match and permission. The match setting is same as
+   the identifier in ACL and permission is the permission to grant, or deny, for this user.
+
+   Example config:
 
    .. code-block:: none
 
-     plugin {
-       # Per-user ACL:
-       acl = vfile
+      namespace inbox {
+         mailbox DUMPSTER {
+            acl owner {
+              rights =
+            }
+         }
+
+         acl user=admin {
+           rights = lr
+         }
+
+         # you can also set id explicitly
+         acl some-name {
+           id = user=foo
+           rights = lr
+         }
+      }
+
+.. dovecot_plugin:setting:: acl_id
+   :plugin: acl
+   :added: 3.0.0
+   :values: @string
+
+   Specifies identity to match. See :ref:`acl-file_format` for values.
+   This is usually used in acl {} block, so the acl\_ prefix can be left out.
```

Comment:
This wasn't added.

Reply:
```
This is usually used in :dovecot_plugin:ref:`acl` {} block,
```

Reply:
That's a bit different. Because 1) usually you'd use it in the `acl id {` filter name, not inside acl {} block. 2) My version says that it's the same setting as the filter name. And anyway 3) I think we should try to keep these texts as consistent as possible. I guess eventually we would move out of Sphinx and make them more of a standard, but for now it would be good so people see similar text in these filter descriptions when it makes sense.